### PR TITLE
test: Added unit test cases for dismiss dialog

### PR DESCRIPTION
**Pull request by devendrafyle**
## Clickup
https://app.clickup.com/t/86cw6epbu

### CODE REVIEW

File: src/app/fyle/dashboard/tasks/dismiss-dialog/dismiss-dialog.component.spec.ts [hunk 1]
```diff
@@ -1,24 +1,100 @@
...
+    popoverController.dismiss.and.callThrough();
+    component.dismissCallInProgress = false;
+    fixture.detectChanges();
+
+    component.cancel();
+    expect(popoverController.dismiss).toHaveBeenCalledTimes(1);
+  });
+
+  describe('dismiss():', () => {
+    it('should dismiss item', (done) => {
```
Comment (Dimple16) [inline src/app/fyle/dashboard/tasks/dismiss-dialog/dismiss-dialog.component.spec.ts:62]: `should dismiss expense?`

File: src/app/fyle/potential-duplicates/potential-duplicates.page.spec.ts [hunk 2]
```diff
@@ -166,38 +175,69 @@ describe('PotentialDuplicatesPage', () => {
...
 
-      expect(expensesService.dismissDuplicates).toHaveBeenCalledOnceWith(
-        ['tx5fBcPBAxLv', 'tx3nHShG60zq'],
-        ['tx5fBcPBAxLv', 'tx3nHShG60zq']
-      );
-      expect(component.selectedSet).toEqual(0);
-      expect(trackingService.dismissedDuplicateSet).toHaveBeenCalledTimes(1);
-      expect(component.showDismissedSuccessToast).toHaveBeenCalledTimes(1);
-      expect(component.loadData$.next).toHaveBeenCalledTimes(1);
+      if (popoverResponse.data?.status === 'success') {
```
Comment (Dimple16) [inline src/app/fyle/potential-duplicates/potential-duplicates.page.spec.ts:235]: shouldn't we test the else case too?

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

File: src/app/fyle/dashboard/tasks/dismiss-dialog/dismiss-dialog.component.spec.ts (hunk 1 above)
Comment (devendrafyle) [reply src/app/fyle/dashboard/tasks/dismiss-dialog/dismiss-dialog.component.spec.ts:62]: yep updated

File: src/app/fyle/potential-duplicates/potential-duplicates.page.spec.ts (hunk 2 above)
Comment (devendrafyle) [reply src/app/fyle/potential-duplicates/potential-duplicates.page.spec.ts:258]: added these cases too

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**